### ISSUE TITLE
Sound Open Firmware (SOF) and Nvidia-Docker

### ISSUE THREAD

**Issue by sudocurse** [sudocurse/reading-list#8181]
Sound Open Firmware (SOF) and Nvidia-Docker<br>
http://ift.tt/2DsKvIy<br>
<ul>
<li>
<h5><a href="https://www.phoronix.com/scan.php?page=news_item&amp;px=Sound-Open-Firmware" rel="nofollow">Intel Open-Sources Sound Firmware, Pushing For More Open Firmware</a></h5>
<blockquote>
<p>Imad Sousou, Intel's GM of the Open-Source Technology Center, had some interesting remarks to make during his keynote today as part of this week's Embedded Linux Conference in Portland.</p>
<p>First up, they have two new open-source project announcements: ACRN and Sound Open Firmware (SOF).</p>
<p>Sound Open Firmware has us most excited with Intel's focus now on opening up more of their firmware, beginning with audio. Sound Open Firmware includes an open-source audio DSP firmware and SDK. The SOF stack works on all Intel hardware platforms and can assist in debugging audio/DSP issues.</p>
</blockquote>
</li>
<li>
<h5><a href="https://thenewstack.io/primer-nvidia-docker-containers-meet-gpus/" rel="nofollow">A Primer on Nvidia-Docker &mdash; Where Containers Meet GPUs</a></h5>
<blockquote>
<p>GPUs are critical for training deep learning models and neural networks. Though it may not be needed for simple models based on linear regression and logistic regression, complex models designed around convolutional neural networks (CNNs) and recurrent neural networks heavily rely on GPUs. Especially computer vision-related models based on frameworks such as Caffe2 and TensorFlow have a dependency on GPU.</p>
<p>In supervised machine learning, a set of features and labels are used to train a model. Deep learning algorithms don&rsquo;t even need explicit features to evolve trained models. They pretty much &ldquo;learn&rdquo; from existing datasets designated for training, testing, and evaluation.</p>
</blockquote>
</li>
</ul>
<p><a href="http://www.tuxmachines.org/node/109847" target="_blank">read more</a></p>
<br>
<br>
via Tux Machines http://ift.tt/VTl7oF<br>
March 14, 2018 at 07:10PM
